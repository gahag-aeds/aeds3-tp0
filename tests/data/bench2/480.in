480 240
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 0 1 44 83 89 98 42 63 38 99 39 55 35 93 7 86 55 53 19 94 31 71 35 51 45 81 65 36 32 98 
Add 121 280
Min 94 253
Add 293 452
Max 258 417
Add 229 388
Sum 122 281
Sub 248 407
Max 272 431
Add 37 196
Max 47 206
Add 23 182
Min 156 315
Sub 122 281
Sum 50 209
Add 169 328
Min 141 300
Sub 136 295
Min 96 255
Sub 6 165
Max 256 415
Sub 142 301
Max 157 316
Add 299 458
Max 18 177
Add 269 428
Min 264 423
Add 259 418
Max 286 445
Sub 312 471
Max 224 383
Add 17 176
Sum 305 464
Add 189 348
Sum 309 468
Sub 114 273
Max 97 256
Sub 302 461
Max 41 200
Sub 240 399
Max 18 177
Add 87 246
Max 134 293
Add 173 332
Max 80 239
Sub 184 343
Min 213 372
Sub 220 379
Sum 319 478
Sub 308 467
Max 226 385
Sub 126 285
Sum 129 288
Sub 62 221
Sum 104 263
Sub 18 177
Min 202 361
Sub 240 399
Min 149 308
Sub 40 199
Sum 103 262
Sub 52 211
Sum 56 215
Add 279 438
Max 112 271
Sub 236 395
Min 72 231
Sub 80 239
Max 217 376
Add 113 272
Sum 192 351
Sub 106 265
Sum 71 230
Add 5 164
Max 278 437
Add 151 310
Max 188 347
Sub 42 201
Sum 50 209
Add 187 346
Sum 29 188
Add 147 306
Sum 312 471
Add 29 188
Sum 81 240
Sub 288 447
Max 47 206
Sub 154 313
Sum 79 238
Add 195 354
Min 193 352
Add 181 340
Min 118 277
Sub 120 279
Max 140 299
Add 101 260
Sum 35 194
Add 83 242
Min 52 211
Add 123 282
Max 195 354
Add 243 402
Max 228 387
Sub 138 297
Min 248 407
Add 57 216
Min 288 447
Add 115 274
Sum 98 257
Add 209 368
Min 173 332
Sub 318 477
Sum 36 195
Add 37 196
Sum 27 186
Sub 308 467
Max 4 163
Add 265 424
Sum 141 300
Sub 82 241
Sum 139 298
Sub 14 173
Sum 134 293
Sub 128 287
Max 5 164
Add 145 304
Min 100 259
Sub 40 199
Max 228 387
Add 151 310
Sum 162 321
Sub 294 453
Min 74 233
Sub 70 229
Sum 111 270
Add 1 160
Max 126 285
Sub 78 237
Sum 307 466
Sub 96 255
Max 159 318
Sub 32 191
Max 93 252
Sub 66 225
Sum 68 227
Add 119 278
Min 53 212
Add 263 422
Max 63 222
Add 193 352
Sum 216 375
Add 73 232
Min 78 237
Add 29 188
Sum 72 231
Add 83 242
Min 174 333
Add 171 330
Max 314 473
Add 81 240
Min 2 161
Add 155 314
Max 54 213
Add 267 426
Sum 96 255
Add 37 196
Max 267 426
Sub 94 253
Sum 306 465
Add 125 284
Max 189 348
Sub 144 303
Sum 156 315
Add 281 440
Max 209 368
Add 95 254
Max 272 431
Add 261 420
Min 37 196
Sub 14 173
Sum 6 165
Sub 252 411
Sum 279 438
Add 275 434
Sum 153 312
Sub 30 189
Min 229 388
Sub 6 165
Sum 73 232
Sub 222 381
Max 278 437
Sub 74 233
Min 249 408
Add 203 362
Max 212 371
Sub 216 375
Min 111 270
Sub 30 189
Sum 309 468
Add 97 256
Max 154 313
Add 177 336
Min 240 399
Sub 182 341
Max 10 169
Add 129 288
Max 276 435
Add 281 440
Max 261 420
Add 185 344
Sum 166 325
Sub 138 297
Min 308 467
Sub 124 283
Min 285 444
Sub 140 299
Max 26 185
Add 65 224
Sum 17 176
Sub 98 257
Max 159 318
Sub 166 325
Min 43 202
Add 87 246
Min 241 400
Sub 126 285
Max 302 461
Add 31 190
Sum 155 314
Add 163 322
Min 127 286
Sub 308 467
Min 211 370
Add 47 206
Min 41 200
Sub 220 379
Max 47 206
Add 189 348
Min 180 339
Add 179 338
Min 53 212
