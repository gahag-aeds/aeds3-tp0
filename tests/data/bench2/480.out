1
100
8388
100
101
0
8205
1
2
101
99
100
0
104
103
8507
8505
99
101
101
100
100
-1
8062
100
7963
7970
-3
-4
7810
7472
95
-4
99
7834
7369
99
99
7391
7309
7933
7734
98
7637
-2
-3
96
7618
-2
101
102
-1
0
8391
1
7964
7957
100
8611
8477
8322
99
-1
102
9001
-1
8257
100
7789
99
99
7302
-4
97
8543
-3
7721
0
104
-1
100
8660
105
8112
105
9407
106
106
0
7549
8526
9486
5
8041
106
4
107
-2
8248
105
4
98
106
107
9807
-4
1
96
7295
104
-4
4
107
9098
-3
4
-2
101
4
-2
